editor-components/feedback/feedbackmodal1/feedbackmodal1.tsx

```diff
@@ -0,0 +1,249 @@
+import * as React from "react";
+import { BaseModal } from "../../EditorComponent";
+import styles from "./feedbackmodal1.module.scss";
+import { ErrorMessage, Formik, Form, Field } from "formik";
+import * as Yup from "yup";
+
+type Emoji = {
+  id: number;
+  label: string;
+  icon: string;
+  color: string;
+};
+
+const emojis: Emoji[] = [
+  { id: 1, label: 'Very Bad', icon: '😢', color: '#E74C3C' },
+  { id: 2, label: 'Bad', icon: '😞', color: '#F39C12' },
+  { id: 3, label: 'Medium', icon: '😐', color: '#F1C40F' },
+  { id: 4, label: 'Good', icon: '😊', color: '#2ECC71' },
+  { id: 5, label: 'Very Good', icon: '😂', color: '#3498DB' },
+];
+
+class FeedbackModal1 extends BaseModal {
+  constructor(props?: any) {
+    super(props, styles);
+
+    this.addProp({
+      type: "string",
+      key: "headerTitle",
+      displayer: "Header Title",
+      value: "How are you feeling?",
+    });
+
+    this.addProp({
+        type:"icon",
+        key:" feedbackIcon",
+        displayer:"Feedback Icon",
+        value:"https://www.google.com/imgres?q=comment%20icon&imgurl=https%3A%2F%2Ficons.veryicon.com%2Fpng%2Fo%2Fhardware%2Fjackdizhu_pc%2Fcomment-25.png&imgrefurl=https%3A%2F%2Fwww.veryicon.com%2Ficons%2Fhardware%2Fjackdizhu_pc%2Fcomment-25.html&docid=fMJtLYW6rlY7AM&tbnid=iadJ3A-hOcv3EM&vet=12ahUKEwjJlcWx8pmHAxXrQ_EDHaERB_cQM3oECBkQAA..i&w=512&h=512&hcb=2&ved=2ahUKEwjJlcWx8pmHAxXrQ_EDHaERB_cQM3oECBkQAA"
```

Comment:
this should not be a link. it should be name of the icon. e.g. : FaArrowDown

[please read](https://github.com/Teknodev/composer-tools/blob/main/editor-components/README.md)